src/test/scala/org/broadinstitute/dsde/workbench/sam/TestSupport.scala

```diff
@@ -116,6 +90,7 @@ object TestSupport extends TestSupport {
   def genSamDependencies(resourceTypes: Map[ResourceTypeName, ResourceType] = Map.empty, googIamDAO: Option[GoogleIamDAO] = None, googleServicesConfig: GoogleServicesConfig = googleServicesConfig, cloudExtensions: Option[CloudExtensions] = None, googleDirectoryDAO: Option[GoogleDirectoryDAO] = None, policyAccessDAO: Option[AccessPolicyDAO] = None)(implicit system: ActorSystem) = {
     val googleDirectoryDAO = new MockGoogleDirectoryDAO()
     val directoryDAO = new MockDirectoryDAO()
+    val registrationDAO = new MockDirectoryDAO()
```

Comment:
Should `MockDirectoryDAO` also extend `RegistrationDAO`?  I know _technically_ this works, but I think we should be explicit about the contract.

Reply:
but DirectoryDAO extends RegistrationDAO... why do we need more?